blocks/core/ff_module/ff_module-profile-response-button/ff_module-profile-response-button.md

```diff
@@ -1,9 +1,32 @@
 <div data-ff_module-profile-response-button=""/>
 
-# Here be a profile-response button
+# Profile response button
 
 ## Markup 
 ```
 <div data-ff_module-profile-response-button=""/>
 ```
+### Props
 
+- `onSelect`: Function
+- `uiState`: String; css defined for `is-selected`, `is-updated` (default), `is-reviewed` in Melody
```

Comment:
My point is that managing CSS state modifiers is something that feels internal to the component. The props should describe the data that needs to be rendered, and the component is responsible for working what UI state that corresponds to. Considering just the data, whether or not this has been selected feels distinct from whether or not it's been (say) reviewed.